[2024-03-09 04:00:09,812] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project492.merge_data 2024-03-09T03:00:00+00:00 [queued]>
[2024-03-09 04:00:09,823] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project492.merge_data 2024-03-09T03:00:00+00:00 [queued]>
[2024-03-09 04:00:09,823] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-03-09 04:00:09,823] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2024-03-09 04:00:09,823] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-03-09 04:00:09,838] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): merge_data> on 2024-03-09T03:00:00+00:00
[2024-03-09 04:00:09,843] {standard_task_runner.py:52} INFO - Started process 26276 to run task
[2024-03-09 04:00:09,846] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'meeting_project492', 'merge_data', '2024-03-09T03:00:00+00:00', '--job-id', '387', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp2gukm1_8', '--error-file', '/tmp/tmpzpd5ii67']
[2024-03-09 04:00:09,848] {standard_task_runner.py:77} INFO - Job 387: Subtask merge_data
[2024-03-09 04:00:09,886] {logging_mixin.py:109} INFO - Running <TaskInstance: meeting_project492.merge_data 2024-03-09T03:00:00+00:00 [running]> on host 62b9685fb4d1
[2024-03-09 04:00:09,936] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=meeting_project492
AIRFLOW_CTX_TASK_ID=merge_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-09T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-09T03:00:00+00:00
[2024-03-09 04:00:09,977] {python.py:151} INFO - Done. Returned value was: None
[2024-03-09 04:00:09,986] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=meeting_project492, task_id=merge_data, execution_date=20240309T030000, start_date=20240309T040009, end_date=20240309T040009
[2024-03-09 04:00:10,018] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-03-09 04:00:10,038] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
